src/main/java/appland/telemetry/Identity.java

```diff
@@ -0,0 +1,78 @@
+package appland.telemetry;
+
+import java.net.NetworkInterface;
+import java.net.SocketException;
+import java.nio.ByteBuffer;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.jetbrains.annotations.NotNull;
+
+import com.intellij.ide.util.PropertiesComponent;
+
+public class Identity {
+  @NotNull protected final static String MachineIdKey = "appland.machine_id";
+  @NotNull protected final static Stream<String> IgnoredMacAddresses =
+      Stream.of("00:00:00:00:00:00", "ff:ff:ff:ff:ff:ff", "ac:de:48:00:11:22");
```

Comment:
Streams a usually only used a temporary objects. At least I never to it :)

You could make this a set instead to simplify the lookup:
```
@NotNull private final static Set<String> IgnoredMacAddresses =
      Set.of("00:00:00:00:00:00", "ff:ff:ff:ff:ff:ff", "ac:de:48:00:11:22");
// ...
       if (IgnoredMacAddresses.contains(mac)) {
          continue;
        }
```